languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/added.txt

```diff
@@ -8752,3 +8753,106 @@ aliançazinha	aliançazinha	NCFS000
 aliançazinhas	aliançazinha	NCFP000
 descompressor	descompressor	NCMS000
 descompressores	descompressor	NCMP000
+home	home	NCMS000
+homes	home	NCMP000
+PIN	PIN	NCMx000
+switch	switch	NCMS000
+Switch	switch	NCMS000
+SWITCH	switch	NCMS000
+switches	switch	NCMP000
+Switches	switch	NCMP000
+SWITCHES	switch	NCMP000
+prompt	prompt	NCMS000
+Prompt	prompt	NCMS000
+PROMPT	prompt	NCMS000
+prompts	prompt	NCMP000
+Prompts	prompt	NCMP000
+PROMPTS	prompt	NCMP000
```

Comment:
I've never used this word in Portuguese, but I'm not sure about its gender. `O prompt` or `a prompt`? Does it even matter? Could also be tagged as `common`. Maybe we should make this a standard for all barbarisms?

Reply:
Always masculine for me, but no harm in `C`ing it.